src/modules/Jobs/jobCreate/containers/JobCreateForm/useJobCreateForm.tsx

```diff
@@ -374,7 +352,7 @@ export const useJobCreateForm = () => {
   };
 
   const openEditQuestionForm = (index: number) => {
-    setEditedQuestion(questions[index]);
+    setEditedQuestion(questions[index] as QuestionReq);
```

Comment:
initialize the state of `editQuestion` to be an empty array and no need to use `as`

Reply:
As I explained above